src/confcom/azext_confcom/docs/v1_json_file_format.md

```diff
@@ -0,0 +1,221 @@
+# v1 `--input`, `-i` json file format
+
+This document provides a comprehensive reference for the JSON configuration format used by the `confcom` extension's `--input`, `-i` flag.
+This schema allows you to define container properties and fragment specifications to be used with `acipolicygen` and `acifragmentgen` including usage with `VN2`.
+
+## Schema Overview
+
+The configuration file uses a structured JSON format with the following main sections:
+
+| Field        | Type    | Required                                      | Description                                                                                   |
+|--------------|---------|-----------------------------------------------|-----------------------------------------------------------------------------------------------|
```

Comment:
Maybe use a table like this for the policy_enforcement_points as an overview, if it makes sense?